src/test/utils/VaultUtils.sol

```diff
@@ -271,6 +313,43 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         assertEq(getToken0Active(), 0);
     }
 
+    function test_advanceWithNoUserDepositedTokens() public {
+        depositToken0();
+        (uint256 reserves0, uint256 reserves1) = getPairReserves();
+
+        vm.startPrank(address(uint160(strategist) + 1));
+        vm.expectRevert("NOT_PARTICIPANT_OR_STRATEGIST");
+        vault.nextEpoch(reserves0, reserves1);
+    }
+
+    function test_advanceWithDepositedToken1() public {
+        vm.stopPrank();
```

Comment:
hm.. this shouldn't be necessary unless we called `startPrank` without calling `stopPrank` previously, but we shouldn't really do that

Reply:
When debugging a failing test, there was a message along the lines of "there is already an active prank", so I added this. It seems to work fine without this now though.